### ISSUE TITLE
Add lua support

### ISSUE THREAD

**Issue by pfebrer** [conda-forge/siesta-feedstock#35]
### Comment:

Is it possible to add lua/flook/flos support in the conda build?

**Comment by jan-janssen** [comment]
@pfebrer I like the suggestion, feel free to open a pull request.

**Comment by pfebrer** [comment]
What would be the best way to do it? Add extra steps in `build.sh` and link statically in the `arch.make` files?

**Comment by zerothi** [comment]
Yes, but, this will still be based on 4.1, we should not do anything else before 5 is released. :)

**Comment by zerothi** [comment]
Well preparation of the 5 release would be OK, but there will be quite some changes in the build system

**Comment by pfebrer** [comment]
Hmm but is version 5 coming soon? What's the harm of adding lua in the current version?

**Comment by zerothi** [comment]
No harm, just saying that you shouldn't spend time on the master, this repo is for stable releases, and the build system for the upcoming 5 is changing alot, and is still changing.

**Comment by pfebrer** [comment]
Solved in #50
